docs/build/Build-for-Linux.md

```diff
@@ -29,7 +29,7 @@ This document assumes 'harmony' as the root directory. _JerryScript_, _libtuv_ a
 You need to install some packages to build IoT.js, as follows;
 
 ```
-sudo apt-get install gyp cmake build-essential valgrind libbluetooth-dev
+sudo apt-get install gyp cmake build-essential valgrind
```

Comment:
I think installing libbluetooth-dev also be removed in tools/apt-get-install-deps.sh 

    sudo apt-get install -q -y \
        cmake g++ valgrind clang-format-3.8 libbluetooth-dev